tests/Fixture/NumberTreesFixture.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         1.2.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Bake\Test\Fixture;
+
+use Cake\TestSuite\Fixture\TestFixture;
+
+/**
+ * NumberTreeFixture
+ *
+ * Generates a tree of data for use testing the tree behavior
+ */
+class NumberTreesFixture extends TestFixture
```

Comment:
Why do we need this fixture?

Reply:
Overall it's a copy of CakePHP's core NumberTreesFixture, but with a change: "name" column now has a length. Since the length attribute is not needed in CakePHP core, and is not covered with tests, it is a subject to change in the future. So I decided to copy it into the repository.

Reply:
I didn't see where the fixture was used. My mistake.